GiniVision/Classes/GiniVisionUtils.swift

```diff
@@ -159,4 +159,25 @@ internal struct Colors {
             alpha: CGFloat(1.0)
         )
     }
-}
\ No newline at end of file
+}
+
+/**
+    Set the status bar style when ViewControllerBasedStatusBarAppearance is disabled.
+ */
+
+internal func setStatusBarStyleIfNeeded(to statusBarStyle: UIStatusBarStyle,
+                                        application: UIApplication = UIApplication.shared) {
+    if !isViewControllerBasedStatusBarStyleEnabled {
```

Comment:
What would happen if you don't perform this check? If it will just not apply, I might consider removing it personally.

Reply:
Umm, you are right. It can be confusing since you need to check the documentation to check that this method does not have effect if `View Controller based status bar` is enabled.
I just add a minor explanation above.